SonarQube.Client/Api/DefaultConfiguration.cs

```diff
@@ -54,5 +54,29 @@ public static RequestFactory Configure(RequestFactory requestFactory)
                 .RegisterRequest<IGetTaintVulnerabilitiesRequest, V8_6.GetTaintVulnerabilitiesRequest>("8.6");
             return requestFactory;
         }
+
+        public static UnversionedRequestFactory ConfigureSonarCloud(UnversionedRequestFactory requestFactory)
+        {
+            requestFactory
+                .RegisterRequest<IGetVersionRequest, V2_10.GetVersionRequest>()
+                .RegisterRequest<IValidateCredentialsRequest, V3_30.ValidateCredentialsRequest>()
+                .RegisterRequest<IGetLanguagesRequest, V5_10.GetLanguagesRequest>()
+                .RegisterRequest<IGetModulesRequest, V5_40.GetModulesRequest>()
+                .RegisterRequest<IGetRulesRequest, V5_50.GetRulesRequest>()
+                .RegisterRequest<IDownloadStaticFile, V5_50.DownloadStaticFile>()
+                .RegisterRequest<IGetProjectsRequest, V6_20.GetProjectsRequest>()
+                .RegisterRequest<IGetPluginsRequest, V6_30.GetPluginsRequest>()
+                .RegisterRequest<IGetPropertiesRequest, V6_30.GetPropertiesRequest>()
+                .RegisterRequest<IGetQualityProfileChangeLogRequest, V6_50.GetQualityProfileChangeLogRequest>()
+                .RegisterRequest<IGetQualityProfilesRequest, V6_50.GetQualityProfilesRequest>()
+                .RegisterRequest<IGetNotificationsRequest, V6_60.GetNotificationsRequest>()
+                .RegisterRequest<IGetRoslynExportProfileRequest, V6_60.GetRoslynExportProfileRequest>()
+                .RegisterRequest<IGetOrganizationsRequest, V7_00.GetOrganizationsRequest>()
+                .RegisterRequest<IGetIssuesRequest, V7_20.GetIssuesRequestWrapper>()
+                .RegisterRequest<IGetHotspotRequest, V8_6.GetHotspotRequest>()
+                .RegisterRequest<IGetTaintVulnerabilitiesRequest, V8_6.GetTaintVulnerabilitiesRequest>();
+
+            return requestFactory;
```

Comment:
We don't currently have any SonarCloud-specific APIs, so this list is just the newest version of each current request implemented for SonarQube.

I created it by copying the entire list from from the SonarQube registration and running the test. Each time it failed because of a duplicate resgistration I deleted the earliest duplicate and ran it again.